docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -76,6 +76,68 @@ Available information returned through the SDK is outlined in the following tabl
 | `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
 | `VersionNotSupported` |  The client library does not support the version of the encrypted token. |
 
+
+
+## Usage for UID2 Sharers
+
+A UID2 sharer is any participant that wants to share UID2s with another participant. Raw UID2s must be encrypted into UID2 tokens before sending them to another participant. For an example of usage, see [com.uid2.client.test.IntegrationExamples](https://github.com/IABTechLab/uid2-client-java/blob/master/src/test/java/com/uid2/client/test/IntegrationExamples.java) (`runSharingExample` method).
+
+The following instructions provide an example of how you can implement sharing using the UID2 SDK for Java, either as a sender or a receiver.
+
+1. Create an IUID2Client reference:
+ 
+   ```java
+   from uid2_client import Uid2Client
+   client = Uid2Client(base_url, auth_key, secret_key)
+   ```
+2. Refresh once at startup, and then periodically (for example, every hour):
+
+   `keys = client.refresh_keys()`
+
+3. Senders: 
+   1. Call the following:
+
+      ```java
+      from uid2_client import encrypt
+      from uid2_client.identity_scope import IdentityScope
+      ```
+      <!-- >NOTE: For EUID depending for which region/identity_scope = IdentityScope.UID2/encrypted_data = encrypt(raw_uid, identity_scope, keys). -->
```

Comment:
this didn't get rendered into a readable text. as we won't mention EUID we can just do

`encrypted_data = encrypt(raw_uid, IdentityScope.UID2, keys)`

Reply:
@sunnywu @atarassov-ttd : I included Sunny's original note re EUID, as a comment in the doc copy, so that whenever this content gets ported to EUID I would hopefully not miss it. It does not render in the UID2 docs site and is not intended to. Andrei do you feel that's a problem? I don't have an EUID version or any Jiras or anything at the moment, no vehicle to stash Sunny's EUID comment so I chose to include it, commented out. Please advise if this is unacceptable. Thx.